builders/build/canonical-contracts.md

```diff
@@ -59,88 +59,88 @@ There are a set of precompiled contracts included on Moonbeam, Moonriver, and Mo
-    |                 [Preimage](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/preimage/Preimage.sol){target=\_blank}                 |             {{networks.moonbeam.precompiles.preimage}}              |
-    |        [OpenGov Tech Committee](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/collective/Collective.sol){target=\_blank}        | {{networks.moonbeam.precompiles.collective_opengov_tech_committee}} |
-    | [Precompile Registry](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/precompile-registry/PrecompileRegistry.sol){target=\_blank} |             {{networks.moonbeam.precompiles.registry}}              |
-    |                        [GMP](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/gmp/Gmp.sol){target=\_blank}                         |                {{networks.moonbeam.precompiles.gmp}}                |
-    |  [XCM Transactor V3](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/xcm-transactor/src/v3/XcmTransactorV3.sol){target=\_blank}   |         {{networks.moonbeam.precompiles.xcm_transactor_v3}}         |
-    |                 [Identity](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/identity/Identity.sol){target=\_blank}                 |             {{networks.moonbeam.precompiles.identity}}              |
+    |                                                                           Contract                                                                            |                               Address                               |
+    |:-------------------------------------------------------------------------------------------------------------------------------------------------------------:|:-------------------------------------------------------------------:|
+    |      [Parachain Staking](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/parachain-staking/StakingInterface.sol){target=\_blank}      |              {{networks.moonbeam.precompiles.staking}}              |
+    |     [Crowdloan Rewards](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/crowdloan-rewards/CrowdloanInterface.sol){target=\_blank}     |             {{networks.moonbeam.precompiles.crowdloan}}             |
+    |             [ERC-20 Interface](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/balances-erc20/ERC20.sol){target=\_blank}              |               {{networks.moonbeam.precompiles.erc20}}               |
+    |    [Democracy [Removed]](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/pallet-democracy/DemocracyInterface.sol){target=\_blank}     |             {{networks.moonbeam.precompiles.democracy}}             |
```

Comment:
It was actually just removed in Moonbase now, we will do a release per network?

Reply:
Actually [as of runtime 2801 it hasn't been removed on Moonbase](https://github.com/moonbeam-foundation/moonbeam/blob/runtime-2801/runtime/moonbase/src/precompiles.rs) yet. I added a spec_version tag so for the next release, i'll be reminded to check it and remove it